packages/fxa-auth-server/lib/metrics/glean/server_events.ts

```diff
@@ -0,0 +1,161 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// AUTOGENERATED BY glean_parser v8.1.1. DO NOT EDIT. DO NOT COMMIT.
```

Comment:
ignorant Q: but did we need/want to commit this?
Or is the autogen comment a bit misleading for our use case?

Reply:
It is a bit unnecessary/misleading for our use case.  We are also committing the generated files for the frontend metrics.